integreat_cms/cms/models/pois/poi.py

```diff
@@ -145,19 +148,36 @@ def get_translation_model() -> ModelBase:
         """
         return POITranslation
 
+    def delete(self, *args: list, **kwargs: dict) -> None:
+        r"""
+        Deletes the poi
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        """
+        if not self.is_used:
+            super().delete(*args, **kwargs)
+        else:
+            logger.debug(
+                "Can't be deleted because this poi is used by an event or a contact"
+            )
```

Comment:
I think for consistency it would make sense to apply the same solution here as in the organisation pr (return whether the poi was deleted) 
Right now, you can see that a success message is shown if you delete a used poi from the poi form, even though this method makes sure that it does not get deleted